domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/PaymentOption.java

```diff
@@ -2,6 +2,7 @@
 
 public enum PaymentOption {
     IMMEDIATELY("Immediately"),
+    BY_SPECIFIED_DATE("By a set date"),
     FULL_BY_SPECIFIED_DATE("By a set date"),
```

Comment:
I suppose this one we are keeping for backward compatibility only? 

Also on `citizen-frontend` we just have `FULL_BY_SPECIFIED_DATE` how that would work with new `BY_SPECIFIED_DATE` during JSON mapping in Request/Response?

Reply:
get on paymentOption will always return BY_SET_DATE

Reply:
yes, that's what I meant and front-end will fail to map this new enum value

Reply:
there is a PR for frontend

Reply:
Great.